packages/flutter/test/rendering/non_render_object_root_test.dart

```diff
@@ -28,20 +26,20 @@ class RealRoot extends AbstractNode {
   @override
   void attach(Object owner) {
     super.attach(owner);
-    child?.attach(owner as PipelineOwner);
+    child.attach(owner as PipelineOwner);
```

Comment:
The two methods above have an `if (child != null)` check. Are those no longer needed then as well?

Reply:
I'll check! I was following the analyzer here. 😅